File: java/src/test/java/codingdojo/CalcStats1Test.java
package codingdojo;

import org.junit.jupiter.api.Test;

import java.util.stream.Stream;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class CalcStats1Test {
    @Test
    void count() {
        Stream<Integer> values = Stream.of(56, 2, 41);
        long count = new CalcStats1(values).count();
        assertEquals(3, count);
    }
}

File: php/test/CalcStats1Test.php
<?php

namespace Tests;

use CodingDojo\CalcStats1;
use PHPUnit\Framework\TestCase;

class CalcStats1Test extends TestCase
{
    public function testCount()
    {
        $values = [56, 2, 41];
        $calcStats1 = new CalcStats1($values);
        $count = $calcStats1->count();
        $this->assertEquals(3, $count);
    }
}
